### DISCUSSION TITLE
Google Ads conversions

### DISCUSSION THREAD

**Discussion by dbzx10299** [johannschopplich/nuxt-gtag#40] [General]
Google recently got rid of universal analytics and uses the gtag.js library with the google tag. The google tag can configure multiple destinations and we just have to use the google gtag script.

Does this also work for sending data to google ads? I noticed this module has to do with google analytics so I would expect the recommended GA events to work but I wanted to check if this is also valid for google ads

```js
// Measure Google Ads conversions
  gtag('event', 'conversion', {
      'send_to': 'AW-YYYYYY/AbC-D_efG-h12_34-567',
      'value': 1.0,
      'currency': 'USD'
  });
```

**Comment by johannschopplich** [comment]
Yes, you can use every Google tag Id with this module, including:
- Google Analytics 4
- Google Ads

So it will work.
